tools/pd-ctl/pdctl/command/unsafe_command.go

```diff
@@ -42,6 +42,7 @@ func NewRemoveFailedStoresCommand() *cobra.Command {
 		Short: "Remove failed stores unsafely",
 		Run:   removeFailedStoresCommandFunc,
 	}
+	cmd.PersistentFlags().Float64("timeout", 0, "timeout in seconds")
```

Comment:
the default value is 0?

Reply:
Thanks for pointing out, it is better to show 600 here which is the "actual" default value.